### ISSUE TITLE
A choseable install directory would be great.

### ISSUE THREAD

**Issue by Rawsted** [cheat-engine/cheat-engine#1502]
If it is installed on my PC some games cry because CE is installed so i tried to install id to my USB but it won´t even ask. a pretty soon edition of CE was installable on USB

**Comment by gendalv** [comment]
yeah, there isn't even a portable version, but they did add adware to the installer...

**Comment by cheat-engine** [comment]
https://cheatengine.org/downloads.php
